src/components/GridTemplates/DetailsPage1/DetailsPage1.stories.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file Details page template #1 story.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { disableCentered, patterns } from '../../../../.storybook';
```

Comment:
For organisational purposes, it might be appropriate to create a new Storybook category specific for 'Layout' - what are your thoughts?

Reply:
Honestly I'm not a fan of another category. Already, the line between "patterns" and "components" is pretty blurry. So then how confusing would "patterns" versus "layouts" be? And these are all components, technically. 😬 I'm almost of the mind that we should remove the categories entirely. They don't serve much of a purpose for someone using the sidebar (just splitting content, taking things out of order, creating categories that could arguably mean multiple things, and apply to the same components in different ways, etc.)

Reply:
Fair enough! Let's reassess the information architecture of these at a later stage

Reply:
Yeah I'm open to two categories but the differences currently seem super subtle 👍 Happy to reevaluate in the future of course